src/schema/me/index.js

```diff
@@ -80,8 +82,8 @@ const Me = new GraphQLObjectType({
         request,
         { rootValue: { meCreditCardsLoader } }
       ) => {
-        return meCreditCardsLoader().then(results => {
-          return results.length > 0
+        return meCreditCardsLoader().then(({ body }) => {
+          return body && body.length > 0
```

Comment:
Curious, was this previously a bug? 

<img width="179" alt="screen shot 2018-09-20 at 3 41 15 pm" src="https://user-images.githubusercontent.com/236943/45850630-a2d4ed80-bceb-11e8-95d1-096b6ac529df.png">

Reply:
We used to have this loader with out `headers: true`, so it returned the body directly (so this would have been an empty array).

I updated it because I thought I needed the headers to handle the relay connection, but ended up not using it in the end. Could change back but I'm not sure there's an argument either way.

Reply:
np! 👍